Eric-ericthelefty/Homework-1/99-bottles-of-beer/app.js

```diff
@@ -0,0 +1,33 @@
+/*
+  Write a script that prints the lyrics to "99 Bottles of Beer on the Wall" in the terminal.
+  If you're unfamiliar with the song, you can [find the lyrics here](http://www.99-bottles-of-beer.net/lyrics.html).
+
+  Be sure that all of your output includes proper spacing (no words or values running into each other).
+
+  Hint: You can make your output appear on multiple lines by using the string \n to insert a line break.
+
+  For an extra challenge, improve this code to print "1 bottle" rather than "1 bottles." (Hint: read up on if/else
+  conditional statements at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else).
+*/
+
+'use strict';
+
+let b = ' bottle of beer on the wall,'
```

Comment:
This is a really clever way to write this code! All your text is together in your code.
In the future, think about giving your variables descriptive names. This would make the console.log statement self documenting.

Reply:
Also, don't forget about spaces! You got them in all the other lines, but here there's no space between the comma at the end and the number that follows.